components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1377,6 +1385,23 @@ private static IFD getFirstIFD(String fname) throws IOException {
     return firstIFD;
   }
 
+  private void initializeReader(IFormatReader r, String file)
+    throws FormatException, IOException
+  {
+    r.setId(file);
+    removeIFDComments(r);
+  }
+
+  private void removeIFDComments(IFormatReader r) {
+    if (r != null && r instanceof MinimalTiffReader) {
+      if (((MinimalTiffReader) r).ifds != null) {
+        for (IFD ifd : ((MinimalTiffReader) r).ifds) {
+          ifd.remove(IFD.IMAGE_DESCRIPTION);
```

Comment:
Are there other TIFF-based readers with would benefit from a similar behavior? A thought is that such a functionality could be implemented upstream in `MinimalTiffReader` using `MetadataLevel` to control how many IFDs get populated by the reader.

Reply:
Definitely worth thinking about for other readers, though I think OME-TIFF is the only where we have (potentially) a very large amount of text in most/all IFDs.

I didn't go down the ```MetadataLevel``` route as only removing ```IFD.IMAGE_DESCRIPTION``` for ```MetadataLevel.MINIMUM``` wouldn't have solved the primary use case here, which was importing into OMERO.  I can see a case for trying to migrate this logic upstream to at least see what breaks, but I'd prefer to do that in a follow-up PR.  Probably each subclass would need to specify a list of tags that are safe to remove (rather than removing only ```IFD.IMAGE_DESCRIPTION``` for every reader).